packages/suite-base/src/util/broadcast/BroadcastLB.ts

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+import { BROADCAST_CHANNEL_NAME } from "./constants";
+import { BroadcastMessageEvent, ChannelListeners } from "./types";
+
+export default class BroadcastLB {
```

Comment:
I'd say to rename to a proper class name like "BroadcastManager"
```suggestion
export default class BroadcastManager {
```

Reply:
Good idea, done.